### PR TITLE
[2201.10.0] Add data.yaml module to the release note

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Purpose
$subject

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -59,6 +59,45 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+
+type DatabaseConfig record {|
+    string dbName;
+    string username;
+|};
+
+public function main() returns error? {
+    // Can read from a file as well.
+    string yamlString = string
+        `
```
Comment (SasinduDilshara) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:85]:
```suggestion
    string yamlString = string `
```
wdyt?

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -59,6 +59,45 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 ## Ballerina library updates
 
+#### `data.yaml` package
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:62]:
```suggestion
### New features

#### `data.yaml` package
```

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -59,6 +59,45 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+
+The [`data.yaml`](https://lib.ballerina.io/ballerina/data.yaml/latest/) package has been introduced to parse YAML to Ballerina `anydata` value with data projection and serialize Ballerina values to YAML.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:64]:
```suggestion
The [`data.yaml`](https://lib.ballerina.io/ballerina/data.yaml/latest/) package has been introduced to parse YAML as Ballerina `anydata` values with data projection and to serialize Ballerina values to YAML format.
```

**Comment by sm1990** [comment]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -72,6 +72,44 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+    DatabaseConfig database;
+|};
+
+type DatabaseConfig record {|
+    string dbName;
+    string username;
+|};
+
+public function main() returns error? {
+    // Can read from a file as well.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:96]:
```suggestion
    // Similar to content read from a YAML file.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 5]
```diff
@@ -72,6 +72,44 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+        host: "localhost"
+        port: 8080
+        remotePorts: [9000, 9001, 9002, 9003]
+        protocol: "http"
+        database:
+          dbName: "testdb"
+          username: "dbuser"
+          password: "dbpassword"`;
+
+    // Based on the expected type, it selectively converts the YAML string to the record type.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:107]:
```suggestion
    // Based on the expected type, it parses the YAML string to selectively construct the record value.
```

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**